Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: build-test-container
```

Comment:
when I run these I get errors like:

```
tests/test_saasherder.py:6: in <module>
    from saasherder import SaasHerder
saasherder/saasherder.py:10: in <module>
    from .config import SaasConfig
E   ValueError: Attempted relative import in non-package
______________________________________________________________________ ERROR collecting tests/test_template.py ______________________________________________________________________
tests/test_template.py:6: in <module>
    from saasherder import SaasHerder
saasherder/saasherder.py:10: in <module>
    from .config import SaasConfig
E   ValueError: Attempted relative import in non-package
```

Reply:
You are probably on Python 2.  I wasn't sure which one to support but both container images in the repo gets built with Py3 so went ahead with it.

Reply:
I don't mind using Python3, but it should be probably introduced in a separate PR and you'll need to fix the Dockerfile to actually use Python3 - otherwise tests fail https://ci.centos.org/view/Devtools/job/devtools-saasherder/4/console

Reply:
@maxandersen @vpavlin Switching back to Py2.